avocado/plugins/assets.py

```diff
@@ -265,10 +265,14 @@ def pre_tests(self, job):
                 # runnable, because the requirements resolution planned is
```

Comment:
I think this comment is not relevant anymore.

Reply:
Good point!

Reply:
BTW, I changed the comment to:

```
# nrunner/resolver based test should describe their requirements
# explicitly using docstring tags.  But, they can still
# benefit from the parsing of "fetch_asset()" calls just like
# the legacy runner.
```

Let me know if you think that's OK.